docs/content/comparisons/druid-vs-sql-on-hadoop.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: doc_page
+---
+
+Druid vs SQL-on-Hadoop (Hive/Impala/Drill/Spark SQL/Presto)
+===========================================================
```

Comment:
I think this grouping is useful, but I would drop "Hive" as even though it's SQL-on-Hadoop it's not really that similar to the others (none of which run through M/R).